packages/sync/src/Modules.php

```diff
@@ -10,24 +10,45 @@
 class Modules {
 
 	private static $default_sync_modules = array(
-		'Automattic\\Jetpack\\Sync\\Modules\\Constants',
-		'Automattic\\Jetpack\\Sync\\Modules\\Callables',
-		'Automattic\\Jetpack\\Sync\\Modules\\Network_Options',
-		'Automattic\\Jetpack\\Sync\\Modules\\Options',
-		'Automattic\\Jetpack\\Sync\\Modules\\Terms',
-		'Automattic\\Jetpack\\Sync\\Modules\\Menus',
-		'Automattic\\Jetpack\\Sync\\Modules\\Themes',
-		'Automattic\\Jetpack\\Sync\\Modules\\Users',
-		'Automattic\\Jetpack\\Sync\\Modules\\Import',
-		'Automattic\\Jetpack\\Sync\\Modules\\Posts',
-		'Automattic\\Jetpack\\Sync\\Modules\\Protect',
-		'Automattic\\Jetpack\\Sync\\Modules\\Comments',
-		'Automattic\\Jetpack\\Sync\\Modules\\Updates',
-		'Automattic\\Jetpack\\Sync\\Modules\\Attachments',
-		'Automattic\\Jetpack\\Sync\\Modules\\Meta',
-		'Automattic\\Jetpack\\Sync\\Modules\\Plugins',
-		'Automattic\\Jetpack\\Sync\\Modules\\Stats',
-		'Automattic\\Jetpack\\Sync\\Modules\\Full_Sync',
+		'Jetpack_Sync_Modules_Constants',
+		'Jetpack_Sync_Modules_Callables',
+		'Jetpack_Sync_Modules_Network_Options',
+		'Jetpack_Sync_Modules_Options',
+		'Jetpack_Sync_Modules_Terms',
+		'Jetpack_Sync_Modules_Menus',
+		'Jetpack_Sync_Modules_Themes',
+		'Jetpack_Sync_Modules_Users',
+		'Jetpack_Sync_Modules_Import',
+		'Jetpack_Sync_Modules_Posts',
+		'Jetpack_Sync_Modules_Protect',
+		'Jetpack_Sync_Modules_Comments',
+		'Jetpack_Sync_Modules_Updates',
+		'Jetpack_Sync_Modules_Attachments',
+		'Jetpack_Sync_Modules_Meta',
+		'Jetpack_Sync_Modules_Plugins',
+		'Jetpack_Sync_Modules_Stats',
+		'Jetpack_Sync_Modules_Full_Sync',
+	);
+
+	private static $legacy_sync_modules_map = array(
```

Comment:
You might want to check if these can be defined using `const` within the namespace (compile time, truly constant) vs a static class property which is mutable and probably has more overhead.